framework/rbac/BaseManager.php

```diff
@@ -190,6 +190,20 @@ public function getRoles()
     }
 
     /**
+     * Returns defaultRoles as array of Role objects
+     * @since 2.0.12
+     * @return Role[] default roles. The array is indexed by the role names
+     */
+    public function getDefaultRoles()
```

Comment:
Should I create a new PR because this one is already merged?
Can you, please, suggest a name for this method?